src/WebJobs.Script.WebHost/Management/MeshServiceClient.cs

```diff
@@ -119,15 +122,15 @@ public async Task CreateBindMount(string sourcePath, string targetPath)
             httpResponseMessage.EnsureSuccessStatusCode();
         }
 
-        private async Task PublishActivities(IEnumerable<ContainerFunctionExecutionActivity> activities)
+        private async Task PublishActivities(ContainerFunctionExecutionActivityRequest activityRequest)
         {
             // Log one of the activities being published for debugging.
-            _logger.LogDebug($"Publishing function execution activity {activities.FirstOrDefault()}");
+            _logger.LogDebug($"Publishing function execution activity {activityRequest.Activities.FirstOrDefault()}");
 
             var operation = new[]
             {
-                new KeyValuePair<string, string>(Operation, "add-fes"),
-                new KeyValuePair<string, string>("content", JsonConvert.SerializeObject(activities)),
+                new KeyValuePair<string, string>(Operation, AddFES),
+                new KeyValuePair<string, string>("content", JsonConvert.SerializeObject(activityRequest)),
```

Comment:
>JsonConvert.SerializeObject(activityRequest) [](http://example.com/codeflow?start=60&length=44)

This would require deserialization updates on the net-core repo, right ?

Reply:
In https://github.com/Azure/azure-functions-docker-private

But please hold off on reviewing this. @mathewc pointed out an alternative fix that I am trying now.